.eslintrc.json

```diff
@@ -169,24 +169,13 @@
                 "*.js"
             ],
             "rules": {
-                "header/header": ["error", "block", [
-                    "",
-                    {"pattern": " \\* Copyright \\(C\\) (\\d+-)?2022  Yomichan Authors"},
-                    " *",
-                    " * This program is free software: you can redistribute it and/or modify",
-                    " * it under the terms of the GNU General Public License as published by",
-                    " * the Free Software Foundation, either version 3 of the License, or",
-                    " * (at your option) any later version.",
-                    " *",
-                    " * This program is distributed in the hope that it will be useful,",
-                    " * but WITHOUT ANY WARRANTY; without even the implied warranty of",
-                    " * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the",
-                    " * GNU General Public License for more details.",
-                    " *",
-                    " * You should have received a copy of the GNU General Public License",
-                    " * along with this program.  If not, see <https://www.gnu.org/licenses/>.",
-                    " "
-                ]]
+                "header/header": [
+                    "error",
+                    "block",
+                    {
+                        "pattern": " \\* Copyright \\(C\\) (\\d+-)?2023  Yomitan Authors(\n \\* Copyright \\(C\\) (\\d+-)?2022  Yomichan Authors)?\n \\*\n \\* This program is free software: you can redistribute it and/or modify\n \\* it under the terms of the GNU General Public License as published by\n \\* the Free Software Foundation, either version 3 of the License, or\n \\* \\(at your option\\) any later version\\.\n \\*\n \\* This program is distributed in the hope that it will be useful,\n \\* but WITHOUT ANY WARRANTY; without even the implied warranty of\n \\* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE\\.  See the\n \\* GNU General Public License for more details\\.\n \\*\n \\* You should have received a copy of the GNU General Public License\n \\* along with this program\\.  If not, see <https://www\\.gnu\\.org/licenses/>\\.\n "
```

Comment:
Windows uses `\r\n` as EOL indicator. Wonder if developers on windows will have problems passing the linter. Darius will put his editor configuration into a [PR](https://github.com/themoeway/yomitan/pull/101/files#r1145575033). Maybe we can force the `\n` EOL in there as well?

Reply:
Oh good catch. The array format seems like it tests for any of `\r|\n|\r\n` between the strings, so this is technically a regression. I'm for forcing that in the editorconfig if possible :pray:

Reply:
Let's do that in editorconfig :+1: